### ISSUE TITLE
Translations for taboos

### ISSUE THREAD

**Issue by derornos** [Kamalisk/arkhamdb-json-data#1229]
https://github.com/Kamalisk/arkhamdb-json-data/blob/master/translations/zh/taboos.json

As above, I would like to update the taboo list for Korean.

But I know that updating the translation doesn't actually show it on the arkhamdb site.

**Comment by derornos** [comment]
https://github.com/Kamalisk/arkhamdb-json-data/blob/master/update_locales.coffee

I'm guessing that the above file will need an update, am I right?

**Comment by zzorba** [comment]
Yeah, the taboo data is not stored in a localized fashion on ArkhamDB at the moment.  Some additions to the website would be needed to support this.
